infrastructure/.terraform-version

```diff
@@ -1 +0,0 @@
-1.4.4
```

Comment:
Did we want to be removing this?

Reply:
Thanks @jackmaloney, the Terraform version is described in providers.tf and the plan executes on Demo without this file, however it's probably a little naive of me to remove it so I'll add it back in.

Reply:
I did notice that removing this file create a conflict anyway. So adding it back in makes sense.